boards/nucleo-wl55jc/Kconfig

```diff
@@ -22,10 +22,16 @@ config BOARD_NUCLEO_WL55JC
     select HAS_PERIPH_TIMER
     select HAS_PERIPH_UART
 
+    select BOARD_HAS_HSE
```

Comment:
Misses `# Clock configuration` comment as seen on other boards

Reply:
Removed as it is defined in the `source "$(RIOTBOARD)/common/nucleo64/Kconfig"`, maybe that gets more explicit later...